### PR TITLE
add string ressources to recipe detail customisation

### PR DESCRIPTION
**Pull request by ThibaultMiam**

### CODE REVIEW

**Comment by ThibaultMiam** [comment]
* **#49** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/49?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ThibaultMiam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailFooter.mdx [hunk 1]
```diff
@@ -149,4 +149,22 @@ public enum class IngredientStatusTypes {
     NO_MORE_TO_ADD,
     REMAINING_INGREDIENTS_TO_BE_ADDED;
 }
-```
\ No newline at end of file
+```
+### Footer ressources
```
Comment (diarmuidmcg) [inline docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailFooter.mdx:153]:
```suggestion
### Footer resources 
```

**Review by diarmuidmcg — COMMENTED** [review]
in english, ressource = resource 
Use reusable function for your Tip

File: docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailOftenDeleted.mdx [hunk 2]
```diff
@@ -222,3 +222,16 @@ data class OftenDeletedProductParameters(
     val chooseProduct: () -> Unit
 )
 ```
+
+### Often deleted Product ressources 
+
+:::tip
+you can replace and reuse  string  ressources if you want to take advantage of our internationalisation system
```
[the remark this answers is not in the document]
Comment (ThibaultMiam) [reply docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailOftenDeleted.mdx:229]: done

**Pull request merged**